doc/source/library_description/packaging.rst

```diff
@@ -2,73 +2,230 @@
 
 Packaging
 #########
-A Python package organizes and structures a Python library, which contains several
-modules and assets such as examples or binary extensions. A Python package
-offers an easy, reliable, and comprehensive way to distribute and install
-a Python library on a variety of platforms and environments.
+
+A Python package organizes and structures a Python library, which contains
+several modules and assets such as examples or binary extensions. A Python
+package offers an easy, reliable, and comprehensive way to distribute and
+install a Python library on a variety of platforms and environments.
+
+.. note::
+
+   If you want to create a new PyAnsys project according to the guidelines
+   presented in the following lines, consider using the `pyansys template`_.
+
+
+Python scripts, modules, sub-packages and packages
+--------------------------------------------------
+
+It is important to understand the difference between Python scripts, modules,
+sub-packages and packages:
```

Comment:
```suggestion
sub-packages, and packages:
```

Reply:
Will pay more attention to this from now on.